src/asn1.ml

```diff
@@ -615,11 +667,263 @@ struct
     map f g @@ sequence2
       (required ~label:"algorithm" Algo.grammar)
       (required ~label:"parameters" DH.Params.grammar)
+
 end
 
 let map_result f = function Result.Ok x -> Result.Ok (f x) | Result.Error _ as r -> r
 let default_result default = function Result.Error _ -> default () | Result.Ok _ as r -> r
 
+module CVC = struct
+  let rsa_oid = Asn.OID.of_string "0.4.0.127.0.7.2.2.2.1.4"
```

Comment:
I'm not sure how much that would improve readability, and I'm a bit apprehensive to use custom operators.